docs/content/articles/cn/madara-beast-article.md

```diff
@@ -0,0 +1,143 @@
+![thee BEEAAST](https://imgur.com/EBwBNnB.jpg)
+
+# 驾驭巨兽 - Madara和Starknet应用链的革新
+
+**2023年7月20日** · 1分钟阅读时间
+
+<font size=5>_探索区块链技术的未来, 从大幅降低成本到个性化控制_</font>
+
+---
+
+## 概述
+
+- Madara是一个高性能的Starknet排序器，提供了创建定制化和高<!--
+  -->效[应用链](https://www.starknet.io/en/posts/ecosystem/the-starknet-stacks-growth-spurt)的<!--
+  -->能力。
+- 通过使用Substrate框架，Madara强化了Cairo VM的能力，从而实现可证明、安全且灵
```

Comment:
是自动换行的吗？貌似在文中会带来短空格，其他地方类似

Reply:
There is a limit of 80 characters per line in markdown, and a space is automatically added between lines in HTML displays,converting newline into space makes sense for english,but not good for chinese characters because we don't use spaces to separate words in chinese,i use:
"<! --
    ->"
to remove the spaces, which passes the markdown-lint, but not the prettier, so when I use prettier to automatic formatting,it becomes what you see now.